src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge17.java

```diff
@@ -0,0 +1,36 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(17)
+public class Challenge17 extends Challenge {
+
+    public Challenge17(ScoreCard scoreCard) {
+        super(scoreCard);
+    }
+
+    @Override
+    public Spoiler spoiler() {
+        return new Spoiler("this is temporary and is not hardcoded anywhere");
```

Comment:
maybe wecan use the encryption stuff we use elsewhere to hide this :)?

Reply:
Ive ended up using the same system challenge 16 uses and generating the secret and replacing it in the .bash_history file. What do you think?